### ISSUE TITLE
Potential NPM Organization change

### ISSUE THREAD

**Issue by AustinPaquette** [paquette/ui#133]
I hate this, because it's already going to be the third one, but here's what happened.

Started with `@paquette.io`, and then decided that was too long. Didn't hear back about `@paquette` because it's taken, but not in use. At least not publicly. Therefore I secured the `@pqt` name and decided to go fill speed ahead.

I just heard back about the `@paquette` name and if everything works out, I may be able to secure it. If that's the case it becomes the obvious choice and (luckily) no one actually uses this package yet apart from me.

TL;DR

`@paquette.io/ui` → `@pqt/ui` → `@paquette/ui`

I would be able to comfortably ensure this would _never_ change in the foreseeable future since the namespace would be identical to the GitHub structure going forward.

The release cycle would remain unchanged and I'd work with NPM to try and migrate everything over depending on what they're capable of. Ideally I could bring the `@paquette.io` org into the migration too. I guess we will see though.

**Comment by AustinPaquette** [comment]
gooooooooot it.

**Comment by AustinPaquette** [comment]
Re: https://github.com/paquette/ui/commit/0c2558b64c4912bdb43d0dac200142bdd96ba340
